substrate/frame/support/src/storage/generator/map.rs

```diff
@@ -196,7 +196,7 @@ where
 		let value = match unhashed::get::<O>(&current_key) {
 			Some(value) => value,
 			None => {
-				log::error!("Invalid translate: fail to decode old value");
+				crate::defensive!("Invalid translate: fail to decode old value");
```

Comment:
I think there can be two cases here: value does not exist or value exists but failed to decode (corrupt).

Reply:
I believe in the case the value does not exist line 194 will return `None`